oak_runtime/src/node/http/mod.rs

```diff
@@ -217,6 +217,10 @@ impl HttpServerNode {
 
 /// Oak Node implementation for the HTTP server.
 impl Node for HttpServerNode {
+    fn node_type(&self) -> &'static str {
```

Comment:
This feels like it may just be an associated constant to the trait? https://doc.rust-lang.org/edition-guide/rust-2018/trait-system/associated-constants.html

Reply:
https://doc.rust-lang.org/error-index.html#the-trait-cannot-contain-associated-constants